### PR TITLE
Added Ciscomeraki

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ X] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cisco-meraki-source.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+For more detailed steps with troubleshooting examples, visit the official documentation [here](https://developer.cisco.com/meraki/api-v1/#!authorization/authorization).
+
+## Create a Cisco Meraki Source
+
+When you create an Cisco Meraki Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
+
+To configure Cisco Meraki Source:
+1. In Sumo Logic, select **Manage Data** > **Collection** > **Collection**.
+1. On the Collectors page, click **Add Source** next to a **HostedCollector**.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cisco-meraki-source.md:67]:
```suggestion
1. On the Collectors page, click **Add Source** next to a **Hosted Collector**.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cisco-meraki-source.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+1. Select Cisco Meraki.<br/> <img src={useBaseUrl('img/send-data/cisco-meraki-sumo-source-select.png')} alt="cisco-meraki-sumo-source-select.png" width="150" />
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cisco-meraki-source.md:68]: I've swapped this out with a cleaner, hi-res image for you. And sized it down to look good.

```suggestion
1. Select Cisco Meraki.<br/> <img src={useBaseUrl('img/send-data/cisco-meraki-sumo-source-select.png')} alt="cisco-meraki-sumo-source-select.png" width="80" />
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
I've swapped out logo with a cleaner, hi-res image for you. Other than that, looks great!

**Comment by JainM6** [comment]
> I've swapped out logo with a cleaner, hi-res image for you. Other than that, looks great!

Thanks Kim.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**